dockerfiles/cli/cli_tests.bats

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env bats
+# Copyright (c) 2012-2017 Red Hat, Inc
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# HOW TO RUN TEST?
+# To run tests, ensure you have:
+#     - installed bats
+#     - running docker (best experience v 1.12 and higher)
+#     - optional - clean docker ('docker ps' and 'docker ps -a' are empty)
+#         
+# Then run command 'bats cli_tests.bats' 
+# from the directory containing these bats tests.
+#
+# Contributors:
+#   Marian Labuda - Initial Implementation
+#
+#
+
+source ../base/scripts/base/startup_funcs.sh
+export ALL_TEST_PATH=$(pwd)/testrun
+export TEST_DIR_PATH=$ALL_TEST_PATH/testdata
+mkdir $ALL_TEST_PATH -p
+
+# set up environment
+setup() {
+  clean_docker
+} 
+
+clean_docker() {
+  if [ "$(docker ps -q | wc -l)" -gt 0 ]; then
+    docker kill $(docker ps -q)
+  fi
+  if [ "$(docker ps -qa | wc -l)" -gt 0 ]; then
+    docker rm $(docker ps -qa)
+  fi
+}
+
+@test "test CLI prompt to provide volume for docker sock" {
+  #GIVEN
+  prompt_substring="-v /var/run/docker.sock:/var/run/docker.sock"
+
+  #WHEN
+  result=$(docker run eclipse/che-cli:nightly start || true)
+
+  #THEN
+  [[ $result == *"$prompt_substring"* ]]
+}
+
+@test "test CLI prompt to provide directory for user data" {
+  #GIVEN
+  prompt_substring="-v <YOUR_LOCAL_PATH>:/data"
+
+  #WHEN
+  result=$(docker run -v /var/run/docker.sock:/var/run/docker.sock eclipse/che-cli:nightly start || true)
+
+  #THEN
+  [[ $result == *"$prompt_substring"* ]]
+}
+
+@test "test CLI 'usage' when running container without command" {
+  #GIVEN
+  output=$(usage || true)
+
+  #WHEN
+  result=$(docker run -v /var/run/docker.sock:/var/run/docker.sock eclipse/che-cli:nightly || true)
+
+  #THEN
+  [[ $result == *$usage* ]]
+}
+
+@test "test 'init' and 'destroy --quiet' with existing dir" {
```

Comment:
Sure thing. It popped in my head that with having all tests in one file it would be huge, several hundreds LoC. I will also move tests from dockerfiles/cli folder to dockerfiles/base/tests folder, it will improve user experience with tests execution. Cli folder was my first choice as a dir for CLI but it makes more sense to have it in scripts folder.